### DISCUSSION TITLE
Custom color code with alpha `bg-[rgba(#ffffff,0.1)]`

### DISCUSSION THREAD

**Discussion by Sariln** [tailwindlabs/tailwindcss#9055] [Ideas]
I use this type of code a lot in SCSS, It easy to set color with alpha to a element. I hope Tailwindcss will support type of this code too. Thank you
Example :
In SCSS `background: rgba(#ffffff,0.1)`
what I hope can have in Tailwindcss ` bg-[rgba(#ffffff,0.1)]`

**Comment by wongjn** [comment]
Closest we have in Tailwind at the moment is the [opacity modifier syntax](https://tailwindcss.com/docs/text-color#changing-the-opacity), so in your case, it would be `bg-white/10`, otherwise more generically, you could use hex alpha value like `bg-[#ffffff1a]`.

**Reply by adamwathan** [reply]
Can also do `bg-[#ffffff]/[0.1]` 👍🏻
